README.md

```diff
@@ -1,2 +1,228 @@
-# clinic-database
-A database schema built from an ERD diagram for a clinic
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Clinic\] ](#-clinic-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [ Future Feature ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Clinic] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Clinic]** is a database of patients with some specifics datas
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This project is built using Sql syntax and Postgresql for database hosting and deployment.
+
+<!-- <details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details> -->
```

Comment:
👍 Good job on the progress you've made so far! However, there are a few more improvements to consider:

- [ ] While you've removed some of the commented-out code as suggested by the previous reviewer, there are still remnants of commented code in your project. It's a good practice to have clean and well-documented code without unused or commented-out sections. Please remove any remaining commented code to ensure your codebase stays clean and maintainable. 👷‍♂️🧹

- [ ] [Optional] Additionally, it's essential to ensure your README is clear and professional. Some sections still contain placeholder text, which might have been part of the initial template. To present your project in the best possible light, please go through your README and remove any placeholder text, keeping it neat and ready for your readers. 📝🧼
![image](https://github.com/patriciachrysy/clinic-database/assets/55840999/97876cf0-3c58-4fdb-89a2-abeefae4ba73)

Keep up the great work in refining your project! 👏🚀